### PR TITLE
Fixes Cyborg Defib units

### PR DESCRIPTION
**Pull request by Poojawa**
:cl: Poojawa
fix: Cyborg defib units are now actually functional
/:cl:

#35727 fixed mounts, but cyborg defibs don't have a defib unit, so they would just break since, y'know, there's no unit for the paddles to be connected to.

I was wondering why the fuck my medihound defibs were broken, then confirmed mediborgs were broken as well and figured I'd fix it here.

### CODE REVIEW

**Comment by AnturK** [comment]
Add `if(req_defib)` to the base one instead please.

**Comment by Poojawa** [comment]
Should be good now, yaya

File: code/game/objects/items/defib.dm [hunk 1]
```diff
@@ -307,6 +307,8 @@
 	check_range()
 
 /obj/item/twohanded/shockpaddles/proc/check_range()
+	if(req_defib)
```
Comment (YPOQ) [inline code/game/objects/items/defib.dm]: !req_defib I think

**Comment by Poojawa** [comment]
Good catch

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**